container.go

```diff
@@ -715,3 +719,21 @@ func (c *Container) addDrive(create bool) (err error) {
 
 	return nil
 }
+
+func (c *Container) removeDrive() (err error) {
+	if c.state.Fstype != "" {
+		virtLog.Infof("Unplugging block device for container %s\n", c.id)
+
+		// Add drive with id as container id
```

Comment:
Comment is not appropriate since you're removing the drive here.

Reply:
fixed